### PR TITLE
Improve styling of MQTT debug info

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change
Improve styling of MQTT debug info

image

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/mqtt-device-debug-info-dialog/dialog-mqtt-device-debug-info.ts [hunk 1]
```diff
@@ -204,6 +211,17 @@ class DialogMQTTDeviceDebugInfo extends LitElement {
...
+          list-style-type: none;
+          margin: 4px;
+          padding-left: 16px;
+        }
+        .entitylistitem {
+          margin-bottom: 12px;
+        }
+        .triggerlistitem {
+          margin-bottom: 12px;
+        }
```
Comment (zsarnett) [inline src/dialogs/mqtt-device-debug-info-dialog/dialog-mqtt-device-debug-info.ts:224]: Any reason not to just have a class for `list-item`?

File: src/dialogs/mqtt-device-debug-info-dialog/dialog-mqtt-device-debug-info.ts [hunk 2]
```diff
@@ -204,6 +211,17 @@ class DialogMQTTDeviceDebugInfo extends LitElement {
         ha-switch {
           margin: 16px;
         }
+        .discoverydata {
```
Comment (zsarnett) [inline src/dialogs/mqtt-device-debug-info-dialog/dialog-mqtt-device-debug-info.ts:214]: we should use a `-` to separate words in class names

**Comment by zsarnett** [comment]
Can we get a before and After?

**Comment by bramkragten** [comment]
@emontnemery asked me to look at the styling, will update this PR later.

**Comment by zsarnett** [comment]
When you comment on a PR but Bram ignores it 😢

**Comment by bramkragten** [comment]
> When you add suggestions on a PR but Bram ignores them 😢 

Haha which one?

**Comment by zsarnett** [comment]
The two Comments above. For class names and the two list item classes

**Pull request merged**